### PR TITLE
blockdev_commit_auto_readonly:Support auto-read-only=on for filesystem backend

### PR DESCRIPTION
**Pull request by aliang123**
Support auto-read-only=on for various backends:filesystem at present

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2000429

### CODE REVIEW

**Comment by aliang123** [comment]
PASS 1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_commit_auto_readonly.q35

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

**Comment by zhencliu** [comment]
Hi @aliang123 , 
From your commit msg: blockdev_commit_auto_readonly:Support auto-read-only=on for filesystem backend
But now we support auto-read-only=on by default, and in your case you explicitly disabled auto-read-only for sn2, so here we are testing the scenario that auto-read-only=off? Can we update commit msg to make it clear?

File: qemu/tests/cfg/blockdev_commit_auto_readonly.cfg [hunk 1]
```diff
@@ -0,0 +1,29 @@
+- blockdev_commit_auto_readonly:
+    type = blockdev_commit_auto_readonly
+    virt_test_type = qemu
+    only Linux
+    only filesystem nbd
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_commit_auto_readonly.cfg:5]: we may remove nbd for this case is for filesystem only

Comment (aliang123) [reply qemu/tests/cfg/blockdev_commit_auto_readonly.cfg]: Remove "only filesystem nbd" to make default "only filesystem". Thanks zhenchao.

**Comment by aliang123** [comment]
> Hi @aliang123 ,
> From your commit msg: blockdev_commit_auto_readonly:Support auto-read-only=on for filesystem backend
> But now we support auto-read-only=on by default, and in your case you explicitly disabled auto-read-only for sn2, so here we are testing the scenario that auto-read-only=off? Can we update commit msg to make it clear?

Zhenchao, for blockdev, auto-read-only is off in default. But in our code, we add cmdline with auto-read-only=on in default. So to meet the scenario, I need to set it off on sn2. Thanks.

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
PASS 1-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.blockdev_commit_auto_readonly

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**